client/blocks/login/login-form.jsx

```diff
@@ -30,13 +34,16 @@ export class LoginForm extends Component {
 		redirectTo: PropTypes.string,
 		requestError: PropTypes.object,
 		translate: PropTypes.func.isRequired,
+		isFormDisabled: PropTypes.bool,
```

Comment:
Yup, that's exactly the difference. `isFormDisabled` is also reused by `GoogleLoginButton`.

I've renamed `isDisabled` to `isDisabledWhileLoading`, to specify what it's used for.